src/Crypto/JWT.hs

```diff
@@ -323,6 +335,18 @@ validateExpClaim conf (ClaimsSet _ _ _ (Just e) _ _ _ _) = do
     else throwError (review _JWTExpired ())
 validateExpClaim _ _ = pure ()
 
+validateIatClaim
+  :: (MonadTime m, HasAllowedSkew a, AsJWTError e, MonadError e m)
+  => a
+  -> ClaimsSet
+  -> m ()
+validateIatClaim conf (ClaimsSet _ _ _ _ _ (Just e) _ _) = do
+  now <- currentTime
+  if (view _NumericDate e) > addUTCTime (abs (view allowedSkew conf)) now
+    then throwError (review _JWTIssuedAtFuture ())
+    else pure ()
+validateIatClaim _ _ = pure ()
+
```

Comment:
I'm uncertain about this change.  There is no requirement or recommendation in https://tools.ietf.org/html/rfc7519#section-4.1.6 regarding the `"iat"` claim.

I am happy for this validation to occur by default, but there should be a way to disable it (perhaps a boolean field in `JWTValidationSettings`?)